packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -19,7 +19,7 @@
           ></span>
         </div>
         <div ref="headerWrapper" :class="['header-wrapper card-title', cardType, {'text-white':!isLightBg}]">
-          <slot name="header"></slot>
```

Comment:
wondering if we should add some opacity transition of sorts to reduce the movement:

![untitled](https://user-images.githubusercontent.com/3306138/101977105-204af680-3c86-11eb-926a-55f921e4802b.gif)

Reply:
instead of removing it completely*

Reply:
I see so maybe like `opacity: 0.5` or something to that effect?

Reply:
Are you referring to this sort of behavior?

![Dec-12-2020 19-52-46](https://user-images.githubusercontent.com/16506539/101983111-ae89a180-3cb3-11eb-9aef-be524fe4f38a.gif)

Reply:
yup - this looks great

so the height of the header shouldn't change before / after expansion, which may be jarring for the reader

Reply:
Okay thanks! I might need some more time to test this out, but I think I have addressed the rest of the PR comments.